src/reportSources/legacyFflogs/__tests__/__snapshots__/eventAdapter.ts.snap

```diff
@@ -158,7 +158,7 @@ Array [
     "targets": Array [
       Object {
         "amount": 36194,
-        "overheal": 0,
+        "overheal": 36194,
```

Comment:
to confirm; this is counting as overheal because it's a single-event test, and ergo is effectively ghosted, as such, is expected?

Reply:
Correct across the board.